src/main/java/uk/gov/justice/digital/service/DomainOperations.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.justice.digital.service;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.slf4j.LoggerFactory;
+import uk.gov.justice.digital.domains.model.TableInfo;
+import uk.gov.justice.digital.domains.model.TableTuple;
+import uk.gov.justice.digital.exceptions.DomainExecutorException;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class DomainOperations {
+
+    protected DataStorageService deltaService = new DataStorageService();
```

Comment:
Since we're using micronaut this should be injected using micronaut's dependency injection.